simple_https_client/src/lib.rs

```diff
@@ -126,10 +126,10 @@ const SHA384_Aes256Gcm_EcdsaSecp256r1Sha256_P256: Algorithms = Algorithms(
 
 pub fn ciphersuites() -> Vec<Algorithms> {
     vec![
-        // SHA256_Chacha20Poly1305_RsaPssRsaSha256_X25519,
-        SHA256_Chacha20Poly1305_EcdsaSecp256r1Sha256_X25519,
-        SHA256_Chacha20Poly1305_EcdsaSecp256r1Sha256_P256,
-        // SHA256_Chacha20Poly1305_RsaPssRsaSha256_P256,
+        SHA256_Chacha20Poly1305_RsaPssRsaSha256_X25519,
```

Comment:
Let's uncomment all of them. They should all be usable now.

Reply:
I've uncommented them and changed the client to try each one until one of them is successful.